mitodl/micromasters
# Added course run and description to exam run

#### What are the relevant tickets?
fixes https://github.com/mitodl/micromasters/issues/4138

#### What's this PR do?
- Adds description field in `ExamRun` model.
- Adds respective course run in `ExamRun` model
- Made fields nullable and black for previous records

#### How should this be manually tested?
- Open admin dashboard and go to http://micromasters.odl.local:8079/admin/exams/examrun/
- You can open existing run and add description or course run
- You can see run in exam run listing
- You can filter existing records with course run

@pdpinch 
#### Screenshots (if appropriate)
<img width="957" alt="screen shot 2018-09-28 at 6 28 33 pm" src="https://user-images.githubusercontent.com/10431250/46211193-61c48500-c34c-11e8-8021-1d547dfe69c9.png">

<img width="164" alt="screen shot 2018-09-28 at 6 28 40 pm" src="https://user-images.githubusercontent.com/10431250/46211194-61c48500-c34c-11e8-8177-7756aa9bc6a9.png">
